traversal/traversal/traversal.py

```diff
@@ -212,7 +214,7 @@ def index() -> rx.Component:
         ),
         bg=page_background,
         height="100vh",
-    )
+    ), rx.color_mode.button(position="top-right")
```

Comment:
nit: Can we put this in the main center? I think in the examples we shouldn't do this tuple return